shell/assets/translations/en-us.yaml

```diff
@@ -1236,7 +1236,7 @@ cluster:
       memory: Memory
       disk: Disk
       image: Image
-      network: 
+      network:
```

Comment:
I had this spacing problem too. The only way I managed to fix it was by editing in github...